src/repositories/repository.py

```diff
@@ -184,6 +185,24 @@ def __init__(self):
         field_dict['id'] = field_dict.pop('news_id')
         super().__init__(news, field_dict, NewsEntity)
 
+    def delete_outdated(self, days_interval=7):
+        bare_query = Query.from_(self._table).delete().where(
+            fn.Now() > fn.Date(self._fields['publish_date'] + Interval(days=days_interval)))
```

Comment:
`fn.Now() - fn.Date(Interval(days=days_interval)) > fn.Date(self._fields['publish_date'])` would be easier to read - to discus.